app/src/main/res/values/strings.xml

```diff
@@ -361,31 +361,31 @@
     <string name="error_no_activity_to_view">Could not find an app to open the content with</string>
     <string name="manage_clients">Manage Clients</string>
     <string name="apps">Apps</string>
-    <string name="generate">Generate</string>
-    <string name="scan">Scan</string>
+    <string name="generate">Generate…</string>
+    <string name="scan">Scan…</string>
     <string name="using_qr_code">Using QR Code</string>
     <string name="authors">Authors</string>
     <string name="author">Author</string>
-    <string name="error_communication_security_credentials">Credentials mismatch (needs pairing)</string>
+    <string name="error_communication_security_credentials">Mismatching credentials (needs pairing)</string>
     <string name="invalidate">Invalidate</string>
     <string name="deny">Deny</string>
     <string name="error_communication_security">A security error occurred</string>
     <string name="nickname">Nickname</string>
     <string name="root_folder">Root</string>
     <string name="add_storage_access">Add Storage Access…</string>
-    <string name="clear_storage_list">Clear Storage List</string>
+    <string name="clear_storage_list">Clear Storage List…</string>
```

Comment:
This is a button that instantly executes (no dialogs & no windows), so I don't think an ellipsis is fitting here.

Reply:
```suggestion
    <string name="clear_storage_list">Clear Storage List</string>
```